devel/fortran-utils/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            fortran-utils
+version         1.2
+revision        0
+categories      devel
+platforms       darwin
+license         BSD
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage        https://sourceforge.net/projects/fortran-utils/
+master_sites    sourceforge
+extract.suffix  .tar.bz2
+worksrcdir      {}
+checksums       rmd160  353e753b2084f3eab5cf9cfd9c2ccf4e983f05b5 \
+                sha256  b2a25ae867a7755fb8d839ca5dd59a24888d12d217aea6ddfd57aceda888b197 \
+                size    7689
+
+description     Tools for use with Fortran77 code
+
+long_description \
+    The fortran-utils are: \
+    \n- fpr: transform Fortran's carriage control conventions to UNIX line printer conventions \
+    \n- fsplit: split a multi-routine Fortran file into individual files
+
+notes "This version has patches from FreeBSD and NetBSD."
+
+supported_archs noarch
+installs_libs   no
+use_configure   no
+build {
+    system -W ${worksrcpath}/fsplit "gcc -o fsplit fsplit.c"
+    system -W ${worksrcpath}/fpr    "gcc -o fpr fpr.c"
+}
```

Comment:
you need to make sure that the port is [UsingTheRightCompiler](https://trac.macports.org/wiki/UsingTheRightCompiler). 

I notice there haven't been any releases since 2012 - is there not an actually maintained package that provides similar functionality?